### DISCUSSION TITLE
Cannot Update Table column label dynamically.

### DISCUSSION THREAD

**Discussion by nandhakumar-rs** [heroui-inc/heroui#613] [Q&A] [answered]
I think the problem is the table header is not re-rendering it seems.

``` 
<Table.Header>  
 ...   
</Table.Header>
```

Any Idea on this?

**Comment by cyanxiao** [comment, marked as answer]
Hey @nandhakumar-rs, Table column label can be updated, please check out [this working sample](https://codesandbox.io/s/sandpack-project-forked-c5tft3?file=/App.js). Could you share your repo or part of the codes so that I can look into it?

**Reply by ali-shafi-hff** [reply]
I am new using Next UI, I have a scenario where I have a Table component and inside the table I have an input/textarea field, if i try to edit the text using arrow keys the navigation takes me to next cell of table.
I have tried using e.preventDefault(); but it doesn't work. how do I disable the arrow key navigation in table and allow user to navigate the text.

[https://github.com/nextui-org/nextui/discussions/3661]
